packages/host/src/Site.tsx

```diff
@@ -31,90 +34,104 @@ const Heading: FC = ({ children }) => (
   </div>
 );
 
-export const Site: FC = () => (
-  <RequireAuthentication>
-    <CommandK>
-      <SnackbarProvider maxSnack={3}>
-        <AppDrawer />
-        <Box flex={1} display="flex" flexDirection="column" overflow="hidden">
-          <AppBar />
-          <Box display="flex" flex={1} overflow="auto">
-            <Routes>
-              <Route
-                path={RouteBuilder.create(AppRoute.Dashboard)
-                  .addWildCard()
-                  .build()}
-                element={<DashboardRouter />}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Catalogue)
-                  .addWildCard()
-                  .build()}
-                element={<CatalogueRouter />}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Distribution)
-                  .addWildCard()
-                  .build()}
-                element={<DistributionRouter />}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Replenishment)
-                  .addWildCard()
-                  .build()}
-                element={<ReplenishmentRouter />}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Suppliers)
-                  .addWildCard()
-                  .build()}
-                element={<Heading>suppliers</Heading>}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Inventory)
-                  .addWildCard()
-                  .build()}
-                element={<InventoryRouter />}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Tools).addWildCard().build()}
-                element={<Heading>tools</Heading>}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Reports)
-                  .addWildCard()
-                  .build()}
-                element={<Heading>reports</Heading>}
-              />
-              <Route
-                path={RouteBuilder.create(AppRoute.Messages)
-                  .addWildCard()
-                  .build()}
-                element={<Heading>messages</Heading>}
-              />
+export const Site: FC = () => {
+  const location = useLocation();
+  const getPageTitle = useGetPageTitle();
+  const { setPageTitle } = useHostContext();
 
-              <Route
-                path={RouteBuilder.create(AppRoute.Admin).addWildCard().build()}
-                element={<Settings />}
-              />
+  useEffect(() => {
+    setPageTitle(getPageTitle(location.pathname));
+  }, [location]);
```

Comment:
This is the only diff in this file. The rest is just indentation/auto-format changes.